_data/shaunferns26/shaunferns26.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shaun
+institution: BITS Pilani
```

Comment:
Use "Birla Institute of Technology and Science, Pilani -- Dubai", fits the 58-character limit mark _perfectly_. If you want you can remove the _Dubai_ if you'd like :D